src/services/category.js

```
@@ -6,9 +6,15 @@ const getCategories = async () => {
 }
 
 const getCategoryById = async (id) => {
-  const category = await Category.findOne({ id })
-  if (!category) throw DOCUMENT_NOT_FOUND('Category')
-  return category
+  if (id.length > 10) {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)